centurio-restapi/src/main/kotlin/fr/hadaly/module/Cors.kt

```diff
@@ -18,6 +18,7 @@ fun Application.corsModule() {
             allowCredentials = true
         } else {
             val frontHost = environment.config.property("ktor.deployment.front_host").getString()
+            host(frontHost, subDomains = listOf("www"))
```

Comment:
Good but you are removing the constraint for the https scheme which is not good idea, you can keep both options.

Reply:
The constraint for https scheme stay present on the next line. In the documentation it seems like we can do this way:
https://ktor.io/docs/cors.html#advanced

Reply:
I don't see that calling two times the method will not override the first call. I think,  the commented lines are there to show some examples.
And even if calling the method was adding each time a new rule, it would mean that the first time you authorize www.HOST for all schemes and second times HOST for https schemes, which in fact would mean that you allow the http scheme for the www subdomain, which is not what we want.
Furthermore, there is no reason to put this on two lines as it concerns the same host.

Reply:
Indeed, it would be override the config. I change that to put on the same line.